src/sentry/seer/code_review/webhook_task.py

```diff
@@ -0,0 +1,124 @@
+from __future__ import annotations
+
+import logging
+from datetime import datetime, timezone
+from typing import Any
+
+import orjson
+from django.conf import settings
+from urllib3.exceptions import HTTPError
+
+from sentry import options
+from sentry.net.http import connection_from_url
+from sentry.seer.signed_seer_api import make_signed_seer_api_request
+from sentry.silo.base import SiloMode
+from sentry.tasks.base import instrumented_task
+from sentry.taskworker.namespaces import seer_code_review_tasks
+from sentry.taskworker.retry import Retry
+from sentry.taskworker.state import current_task
+from sentry.utils import metrics
+
+logger = logging.getLogger(__name__)
+
+
+# This needs to match the value defined in the Seer API:
+# https://github.com/getsentry/seer/blob/main/src/seer/routes/codegen.py
+SEER_PR_REVIEW_RERUN_PATH = "/v1/automation/codegen/pr-review/rerun"
```

Comment:
This matches https://github.com/getsentry/seer/pull/4173/files#diff-d022a5605fc7126215740d58900ce6e7c4be1899c6b4809c8e353bda0883a35dR132